lib/models/emailAction.js

```diff
@@ -50,7 +49,7 @@ function SendMail(action, event, callback) {
 
         metrics.IncMetrics(event.service, event.subservice, metrics.actionEmail);
 
-        transporter.sendMail(mailOptions, function(err, info) {
+        transporter.sendMail(mailOptions, (err, info) => {
```

Comment:
I understand the preferred code style is currently `function(error,info)` over `(err, info) =>`, so this change probably should be rolled-back.

Reply:
Fixed in 9c80eb9